oem/system.go

```diff
@@ -76,3 +76,19 @@ type MemorySummary struct {
 type StatusMemory struct {
 	HealthRollup string `json:"HealthRollup"`
 }
+
+// SystemFirmwareInventory is the json object for SystemFirwmareInventory metadata
+type SystemFirmwareInventory struct {
+	Current []struct {
+		Component []struct {
+			Details []struct {
+				Item []struct {
+					Name     string `json:"Name,omitempty"`
+					Key      string `json:"Key,omitempty"`
+					Location string `json:"Location,omitempty"`
+				}
+				VersionString string `json:"VersionString,omitempty"`
+			}
+		}
+	}
+}
```

Comment:
@jenniferKaiser21 Same on this one as well. It is not used currently and I think it can be removed.

Reply:
@ibrahimkk-moideen Validated this is no longer necessary. I've removed it from the code.